src/main/java/lux/fm/bookingservice/service/impl/BookingServiceImpl.java

```diff
@@ -120,6 +132,14 @@ public void deleteBookingById(Authentication authentication, Long id) {
         bookingRepository.delete(booking);
     }
 
+    private void expireStripeSession(Booking booking) {
+        try {
+            Session.retrieve(booking.getPayment().getSessionId()).expire();
+        } catch (StripeException e) {
+            throw new BookingException("Unsuccessful try to expire stripe session", e);
+        }
```

Comment:
Move this method inside StripeService. And make it with next signature:
`public void expireSession(String sessionId)`
Because I need use it in other place